source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidJavaConverter.cs

```diff
@@ -210,6 +210,29 @@ internal static List<Participant> ToParticipantList(AndroidJavaObject turnBasedM
             }
         }
 
+        internal static List<string> ToStringList(AndroidJavaObject list)
+        {
+            int size = list.Call<int>("size");
+            List<string> converted = new List<string>(size);
```

Comment:
We should handle empty list case

Reply:
Empty list already returns an empty list. 

But I've added handling null case. Returning an empty list is more meaningful than null in this context as it's used for (dis)connected peers.